packages/contracts-bedrock/scripts/ScriptExample.s.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
```

Comment:
I think it could be good to include this in a `testdata` directory in `op-chain-ops` with a go build directive to build the artifact and commit the artifact in

Reply:
done